fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/InsertIntoTableCommand.java

```diff
@@ -152,7 +163,15 @@ public void run(ConnectContext ctx, StmtExecutor executor) throws Exception {
         }
 
         OlapTableSink sink = ((OlapTableSink) planner.getFragments().get(0).getSink());
-
+        if (ctx.getSessionVariable().enableInsertGroupCommit) {
+            // group commit
+            if (!analyzeGroupCommit(sink, physicalOlapTableSink)) {
```

Comment:
Non group commit insert shoud also execute successfully in Nereids, so the `AnalysisException` should not be thrown?
```
if (analyzeGroupCommit(sink, physicalOlapTableSink)) {
    handleGroupCommit(ctx, sink, physicalOlapTableSink);
    return;
}
```